src/main/java/edu/unc/lib/boxc/migration/cdm/InitializeProjectCommand.java

```diff
@@ -53,6 +53,9 @@ public class InitializeProjectCommand implements Callable<Integer> {
                 "If not specified, then the project will be initialized in the current directory.",
                 "If the project name is different from the CDM collection ID, then use -c to specify the ID." })
     private String projectName;
+    @Option(names = {"-s", "--source"},
+            description = {"Initialize a new project that doesn't need to be linked with a CDM collection."})
+    private String newProject;
```

Comment:
The idea with this parameter is that it specifies the source of the data being migrated. So for the time being it would take values like "files" and "cdm", and later it might take "aspace". So I think we should adjust the description and the variable name accordingly, maybe projectSource or something with source in it. And it would default to "cdm". We should also validate its value.